dist/video-stream-merger.js

```diff
@@ -1,14 +1,19 @@
 (function(f){if(typeof exports==="object"&&typeof module!=="undefined"){module.exports=f()}else if(typeof define==="function"&&define.amd){define([],f)}else{var g;if(typeof window!=="undefined"){g=window}else if(typeof global!=="undefined"){g=global}else if(typeof self!=="undefined"){g=self}else{g=this}g.VideoStreamMerger = f()}})(function(){var define,module,exports;return (function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
-/* globals window, AudioContext */
+/* globals window */
 
 module.exports = VideoStreamMerger
 
-window.AudioContext = window.AudioContext || window.webkitAudioContext
-
 function VideoStreamMerger (opts) {
   var self = this
   if (!(self instanceof VideoStreamMerger)) return new VideoStreamMerger(opts)
 
+  var AudioContext = window.AudioContext || window.webkitAudioContext
+  var audioSupport = !!(AudioContext && (new AudioContext()).createMediaStreamDestination)
```

Comment:
The AudioContext constructor can only be called once. Could you change this to `!!(AudioContext && (self._audioCtx = new AudioContext()).createMediaStreamDestination)` and remove the corresponding line below.